### PR TITLE
Fix (Select v18): overlap between outline and border with invalid state

### PR DESCRIPTION
**Pull request by jicedtea**
Fix Select component
Before
image
After
image

### CODE REVIEW

**Review by SoyDiego — APPROVED** [review]

File: src/app/components/select/style/selectstyle.ts [hunk 1]
```diff
@@ -197,8 +197,7 @@ input.p-select-label {
 
 .p-dropdown.ng-invalid.ng-dirty,
 .p-select.ng-invalid.ng-dirty {
-    outline: 1px solid ${dt('selectbutton.invalid.border.color')};
-    outline-offset: 0;
+    border-color: ${dt('selectbutton.invalid.border.color')};
```
Comment (naichinger) [inline src/app/components/select/style/selectstyle.ts:200]: I believe the variable name is also wrong. Instead of "selectbutton" it should be "select".

**Comment by cetincakiroglu** [comment]
Fixed in https://github.com/primefaces/primeng/commit/bf8423bbc53a02f050846e79bae1dc02cd1ad5a8

**Comment by jicedtea** [comment]
> Fixed in [bf8423b](https://github.com/primefaces/primeng/commit/bf8423bbc53a02f050846e79bae1dc02cd1ad5a8)

Still a bug in v18-beta 3

https://v18.primeng.org/select#invalid

Select
image
InputText
image

**Comment by jicedtea** [comment]
Issue opened: https://github.com/primefaces/primeng/issues/16577

**Pull request closed without merging**